public/style.css

```diff
@@ -1,57 +1,81 @@
+body {
```

Comment:
What's up with two `style.css` files? I think you probably want to add another public Express directory using `express.static()` and point it to `client/dist/`

Reply:
this is a result of the eslint webpack plugin,  I will remove these files from the repo and add to gitignore